content/articles/2021sy/blog/jisyukenkyu.md

```diff
@@ -0,0 +1,17 @@
+Title: 自主研究奨励事業2020
+Date: 2021.05.06
+Modified: 2021.05.06
+Tags: Machine Learning
+Authors: 安部,梅田,富田
+
+学部学生による自主研究奨励事業、全学選抜自主研究成果発表会
+
+2021/04/30に銀杏会館でとりおこなわれた発表会にて優秀賞をいただきました。
+![写真]({attach}./images/jisyukenkyu_figs/8519.jpg)
```

Comment:
The image size is large, so you should use 
`<img src="{attach}./images/jisyukenkyu_figs/8519.jpg" alt="jisyukenkyu_figs8519" width="400px">`